### ISSUE TITLE
XSS on the old Wayback interface

### ISSUE THREAD

**Issue by arquivo** [arquivo/pwa-technologies#63]
Originally reported on Google Code with ID 64
```
Reported by simaofontes

 Encontrei mais uma vulnerabilidades Cross Site Scripting.
 Aqui esta o link da nova e da antiga,provavelmente o da antiga não estava funcionando
pois o link trazia alguns erros:

 http://arquivo.pt/wayback/wayback/query?type=urlquery&multDet="><script>alert('xss')</script>&url=http://&Submit=Search

 http://arquivo.pt/wayback/wayback/query?type=urlquery&multDet=true&url="><script>alert('XSS');</script>&date=&Submit=Search

This issue occurs in the old Wayback interface that is being discontinued.
The user-inputed query is naively inserted in the <input name="multDet"> input field.
```

Reported by `devel.david@vcruz.net` on 2013-03-05 17:32:34

**Comment by danielbicho** [comment]
Verified.

The vunerability still exist. Chrome Browser mitigate the attack but in other browsers not.

**Comment by danielbicho** [comment]
Fix at Pull Request #155
